maintainer/conda/MDAnalysis/meta.yaml

```diff
@@ -0,0 +1,76 @@
+package:
+  name: mdanalysis
+  # This has to be changed after a release
+  version: "0.14.0"
```

Comment:
Who is editing `change_relesase,sh`? That's the only thing missing before merging.